packages/fluentui/react-northstar/src/themes/teams/components/Dropdown/dropdownStyles.ts

```diff
@@ -36,22 +37,30 @@ const getWidth = (p: DropdownPropsAndState, v: DropdownVariables): string => {
   return v.width;
 };
 
+const isEmpty = prop => {
+  return (
+    typeof prop === 'object' &&
+    //@ts-ignore
```

Comment:
Is this `ts-ignore` intent? 

I think the non-null assertion is miss used here

Reply:
thank you this good catch I forgot to remove "ts-ignore", leftover...